### ISSUE TITLE
Uploaded item title field has i18n garbage

### ISSUE THREAD

**Issue by cbeer** [sul-dlss/dlme#758]
`Curation > Add Items > Upload item`

<img width="493" alt="Screen Shot 2019-11-20 at 13 02 10" src="https://user-images.githubusercontent.com/111218/69278372-0af4bd00-0b97-11ea-852f-1ac4ff393a80.png">

**Comment by jkeck** [comment]
FWIW, if anybody is looking into this, I've poked a bit at this and have a possible fix.  Would be happy to pair up on this on Monday (or happy to defer if somebody has a fix or we roll w/ what's in the linked PR).

**Comment by cbeer** [comment]
I lightly tweaked @camillevilla's PR earlier today with something that seems to work: https://github.com/sul-dlss/dlme/pull/760

**Comment by jkeck** [comment]
There is still some code in Spotlight that should/could handle this case more appropriately.  We can prob. ship both.

**Comment by cbeer** [comment]
Agreed  -- although it's not clear  to  me what the right fix is .. possibly just  hard-coding the title translation?

**Comment by jkeck** [comment]
Hmm, well, the thing that I had tracked down was some Spotlight code that assumed the title configuration would always be a symbol/string and was not permissible to a field configuration object (although I'm sure there are lots of places that we could touch to address this in one way or another).

My main hold-up is the testing for this bit of code is pretty convoluted and I got bogged down on Friday trying to untangle it.  Glad that we'll have a fix for this now and I can work on untangling the testing.
